src/test/java/ch/wisv/areafiftylan/integration/UserRestIntegrationTest.java

```diff
@@ -25,9 +25,11 @@
 import io.restassured.http.ContentType;
 import io.restassured.http.Header;
 import org.apache.http.HttpStatus;
+import org.apache.tomcat.jni.Local;
```

Comment:
What's this?

Reply:
I don't know. Probably a wrong import it was unused. Removed it!